io/net/sriov_device_test.py.data/README.txt

```diff
@@ -6,6 +6,7 @@ hmc_username = Specify the HMC user name.
 hmc_pwd = Specify the HMC password.
 sriov_adapter = Specify the sriov adapter loc code for test.
 sriov_port: specify adapter port for test.
+sriov_roce: specify if the roce adapter VF needs to be added and removed ex: "1"
```

Comment:
@PavamanSubramaniyam Please make sure it is boolean like True or False instead numeric

Reply:
I have made the changes as suggested.

Reply:
I tested this, but fails.. as zip function does not iterate boolean values.. so better to have string or numeric